include/Ardrivo/Stream.h

```diff
@@ -1,6 +1,6 @@
 /*
- *  Stream.h
- *  Copyright 2020-2021 ItJustWorksTM
+ *  Stream.hpp
+ *  Copyright 2021 ItJustWorksTM
```

Comment:
As far as I'm aware, this file hasn't changed name, and 2020 hasn't been cancelled out of existence

Reply:
sorry, will fix this